.github/workflows/pester.yml

```diff
@@ -1,15 +1,14 @@
-name: Ubuntu_Pester
+name: Pester Tests
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+on: [push]
```

Comment:
Since this is an open repo, and you seem to do all your work via pull requests, I recommend that you remove push, and have the trigger be a pull request. That will make it much easier for open source development, because if you fork the repo the tests will never be triggered since there will not be a push to upstream. For that reason I recommend PR as the trigger

Reply:
nope, good catch. Fixed